### PR TITLE
Generate password

### PR DESCRIPTION
**Pull request by mjmenger**
generate the password to be used for the bigip admin password and return it from the module

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -15,7 +15,7 @@ output "mgmt_port" {
 
 # BIG-IP Password
 output "password" {
-  value = ""
+  value = "${random_password.bigippassword.result}"
```
Comment (aknot242) [inline outputs.tf]: value = "${random_password.bigip_password.result}"

File: random.tf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+resource "random_password" "bigippassword" {
```
Comment (aknot242) [inline random.tf]: Can you make this "bigip_password"?

**Review by aknot242 — CHANGES_REQUESTED** [review]
Suggesting a variable name change to adhere to the convention you have established

File: outputs.tf (hunk 1 above)
Comment (mjmenger) [reply outputs.tf]: I've committed a change to the pull request which addresses your request

File: random.tf (hunk 2 above)
Comment (mjmenger) [reply random.tf]: I've committed a change to the pull request which addresses your request

**Review by jwheelwr — APPROVED** [review]
Looks good!

**Comment by mjmenger** [comment]
based upon a conversation with @codygreen we're removing the password generation from the module. it will be passed through an out-of-band secrets management path.

**Pull request closed without merging**